src/Gui/Navigation/NavigationStyle.cpp

```diff
@@ -1930,6 +1932,8 @@ void NavigationStyle::openPopupMenu(const SbVec2s& position)
         QAction *item = navMenuGroup->addAction(name);
         navMenu->addAction(item);
         item->setCheckable(true);
+        QByteArray data(style.first.getName());
+        item->setData(data);
```

Comment:
```suggestion
        item->setData(QByteArray(style.first.getName()));
```

Or if that does not work it should be named with something less generic than `data`, for example `nameByteArray`